ui/webui/src/app.jsx

```diff
@@ -18,11 +18,19 @@ import cockpit from "cockpit";
 import React, { useEffect, useState } from "react";
 
 import {
-    Page,
+    Page, Wizard,
 } from "@patternfly/react-core";
 
 import { InstallationLanguage } from "./InstallationLanguage.jsx";
-import { Summary } from "./Summary.jsx";
+
+import { InstallationDestination } from "./storage/InstallationDestination.jsx";
```

Comment:
Please not line breaks between imports of the same type. 

I usually do it like this:

First import external libraries, see React.
new line
Then import externally defined components, see react-core
new line
Then import locally defined components
new line

Reply:
ACK, fixed.